app/stylesheets/main.scss

```diff
@@ -126,56 +139,92 @@ body, html {
     color: var(--sn-stylekit-info-color);
   }
 
+  code,
   pre {
-    background: var(--sn-stylekit-contrast-background-color);
-    color: var(--sn-stylekit-contrast-foreground-color);
-    border: 1px solid var(--sn-stylekit-contrast-border-color);
-    padding: 20px;
-    border-radius: 3px;
-
-    overflow: auto;
+    font-family: var(--sn-stylekit-monospace-font);
+  }
 
-    // We tried adding this to make code block wrap if too long, but this destroys spacing of code.
-    // We went with overflow: auto instead to make it scroll
-    // white-space: normal;
-    // word-break: break-all;
+  code {
+    box-sizing: border-box;
+    word-wrap: break-word;
+    padding: 0.2em 0.4em;
+    margin: 0;
+    font-size: 85%;
+    background-color: var(
+      --sn-stylekit-contrast-background-color
+    ); // rgba(27,31,35,.05);
+    border-radius: 3px;
   }
 
-  pre code {
-    background-color: transparent;
+  pre {
+    padding: 16px;
+    overflow: auto;
+    font-size: 85%;
+    line-height: 1.45;
+    background-color: var(
+      --sn-stylekit-contrast-background-color
+    ); // rgba(27,31,35,.05);// #f6f8fa; the latter doesn't work well on dark mode
+    border-radius: 3px;
+    word-wrap: normal;
+    margin-top: 0;
+    margin-bottom: 16px;
     color: var(--sn-stylekit-contrast-foreground-color);
   }
 
-  p code {
-    padding: 2px 6px;
-    background-color: var(--sn-stylekit-contrast-background-color);
-    color: var(--sn-stylekit-info-color);
-    border: 1px solid var(--sn-stylekit-contrast-border-color);;
-    border-radius: 3px;
+  pre > code {
+    padding: 0;
+    margin: 0;
+    font-size: 100%;
+    word-break: normal;
+    white-space: pre;
+    background: transparent;
+    background-color: transparent;
+    border: 0;
   }
 
-  code {
-    font-family: var(--sn-stylekit-monospace-font);
+  pre code {
+    display: inline;
+    max-width: auto;
+    padding: 0;
+    margin: 0;
+    overflow: visible;
+    line-height: inherit;
+    word-wrap: normal;
+    background-color: initial;
+    border: 0;
+    //background-color: transparent;
+    color: var(--sn-stylekit-contrast-foreground-color);
   }
 
   table {
     display: block;
-    margin-bottom: 12px;
     width: 100%;
     overflow: auto;
+    margin-top: 0;
+    margin-bottom: 16px;
+    border-spacing: 0;
     border-collapse: collapse;
-    border-spacing: 0px;
-    border-color: var(--sn-stylekit-contrast-border-color);
-    background-color: var(--sn-stylekit-contrast-background-color);
     color: var(--sn-stylekit-contrast-foreground-color);
 
-    th, td {
+    tr {
+      background-color: transparent; // #fff;
+      border-top: 1px solid var(--sn-stylekit-contrast-border-color); // #c6cbd1;
+    }
+
+    td,
+    th {
       padding: 6px 13px;
-      border: 1px solid var(--sn-stylekit-contrast-border-color);
+      border: 1px solid var(--sn-stylekit-contrast-border-color); // #dfe2e5;
```

Comment:
Comments describing the value of a variable are dangerous. These variables can change at any time at which point these comments will be outdated.

Reply:
OK 👍